app/controllers/carts_controller.rb

```diff
@@ -5,6 +5,11 @@ class CartsController < ApplicationController
 
   def show
     flash[:primary] = "Your cart is empty." if @cart.total_count == 0
+    set_cart
+    @items = @cart.contents.map do |item_id, quantity|
```

Comment:
This gets me all the items in the cart so that I can iterate through them in the cart show page and pull info, like item.name, etc